test/api/cpp/proj-issue656.cpp

```diff
@@ -0,0 +1,28 @@
+#include <cvc5/cvc5.h>
```

Comment:
I believe this should have a cvc5 header like the other unit tests in this directory.